packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -76,6 +76,7 @@ const BLOCK_EDITOR_SETTINGS = [
 	'__unstableIsBlockBasedTheme',
 	'__experimentalArchiveTitleTypeLabel',
 	'__experimentalArchiveTitleNameLabel',
+	'__experimentalGetPostLinkProps',
```

Comment:
Why is this "__experimental" ? It should be either "private" or stable.

Reply:
@youknowriad Do you know if there's prior art for making these private?

I saw in the block editor package there are some private settings, but not so much in the editor package.

Reply:
I'm not sure there's for the EditorProvider yet.

Reply:
I think there's something unusual going on here anyway, as it looks like `getPostLinkProps` is already passed in as an editor setting, so I'm not sure if `__experimentalGetPostLinkProps` is also needed.

Reply:
PR here - https://github.com/WordPress/gutenberg/pull/57535